modules/client/test/integration/tokenVoting-client/methods.test.ts

```diff
@@ -213,8 +213,8 @@ describe("Token Voting Client", () => {
           expect(typeof proposalId).toBe("string");
           expect(proposalId).toMatch(/^0x[A-Fa-f0-9]{40}_0x[A-Fa-f0-9]{1,64}$/);
 
-          const params: ICanVoteParams = {
-            address: TEST_WALLET_ADDRESS,
+          const params: CanVoteParams = {
+            voterAddressOrEns: TEST_WALLET_ADDRESS,
             proposalId,
             vote: VoteValues.YES,
```

Comment:
I think this was already discussed elsewhere, but it's weird to be asking the vote here. I guess it makes sense for vote replacement-related scenarios?

Reply:
Yep, its weird but the contracts ask for the vote